pkg/agent/metrics/prometheus.go

```diff
@@ -20,55 +20,74 @@ import (
 	"k8s.io/klog"
 )
 
+const (
+	metricNSAntrea = "antrea"
+	subsystemAgent = "agent"
```

Comment:
can we use `metricNamespaceAntrea` and `metricSubsystemAgent` instead?

same for the others

Reply:
Updated.